src/App.css

```diff
@@ -382,14 +419,20 @@ label.hover {
 }
 /* placeholder css to test button style */
 .form-button label {
-  display: inline-block;
-  background-color: #ddd;
   margin: 10px;
-  padding: 10px 20px;
-  font-size: 16px;
-  border-radius: 4px;
 }
 
-.form-button label:hover {
-  background-color: #878787;
+.form-button label.bg-light-green:hover {
```

Comment:
I like that these buttons change color slightly and a border shows up on hover to add emphasis.  It was mentioned earlier that it would be nice to have something that lets a user know which button was clicked.  I found this [article ](https://medium.com/claritydesignsystem/pure-css-accessible-checkboxes-and-radios-buttons-54063e759bb3)on focused and check input css but I couldn't get it to work like I thought it should.  We could make a new issue to refactor that.

Reply:
Thanks for linking a related article @meganmckissack 💪